### PR TITLE
Feat/hng 16 implement login page

### PR DESCRIPTION
**Pull request by seshto13**
# Description
Implement the login page 

**Closes #issue number 16**

# Changes proposed

## What were you told to do?
To implement the login page following the design on the figma

## What did you do?
I implement the login page that align with the provided figma design. I ensure that it was responsive and the email and password is validated

# Check List (Check all the applicable boxes)

🚨Please review the [contribution guideline](CONTRIBUTING.md) for this repository.

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title and description of the PR is clear and explains the approach.
- [x] I am making a pull request against the **dev branch** (left side).
- [x] My commit messages styles matches our requested structure.
- [x] My code additions will fail neither code linting checks nor unit test.
- [x] I am only making changes to files I was requested to.

# Screenshots/Videos

### CODE REVIEW

File: .gitignore
Comment (shalomtaiwo) [inline .gitignore:1]: Please don't add the lock files into gitignore. Fix it

**Review by shalomtaiwo — CHANGES_REQUESTED** [review]

File: .gitignore
Comment (seshto13) [reply .gitignore:1]: Ok will fix that now

**Comment by mrcoded** [comment]
Please edit your PRs to contain why you are making the PR @seshto13

**Comment by seshto13** [comment]
> Please edit your PRs to contain why you are making the PR @seshto13

I have edited my PR . kindly check

File: .gitignore
Comment (seshto13) [reply .gitignore:1]: I have fixed the issued

**Review by shalomtaiwo — APPROVED** [review]

**Pull request merged**